requirements.txt

```diff
@@ -1 +1,8 @@
+ansible-lint
+black
+docker
+flake8
+molecule
```

Comment:
Since these are dependencies for CI, and breaking updates like the one of molecule may happen again, I would consider preventing upgrades of major versions here (e.g. using `molecule~=3.0`, ...).
For code style, you may want to be even stricter and limit down to the minor version - I don't know about `black`, but `yapf` keeps changing things.

I used molecule in the CI setup of many ansible roles, the new molecule version does not add anything I need, and I'm still a bit upset that I now need to go through all these roles and fix the tests because of this upgrade.

Reply:
Black is pretty good, and I'd argue that we want to just keep up to date on all formatters to keep the code clean at the cost of later nitpicky updates.

Will pin docker and molecule since those _actually_ run the tests and can break stuff...